### ISSUE TITLE
Cyrrilic text in attached console output in wrong encoding

### ISSUE THREAD

**Issue by f-z-f** [unickq/allure-nunit#18]
Nuget packages used:
NUnit 3.10.1
NUnit.Allure 1.0.5
Allure.Commons 2.3.64.1

Example code:
```
using System;
using NUnit.Framework;
using NUnit.Allure.Core;

namespace AllureTest
{
    [AllureNUnit]
    [TestFixture()]
    public static class Test
    {
        [Test()]
        public static void TestCase()
        {
            Console.WriteLine("ПРОВЕРКА");
            Assert.Pass();
        }
    }
}
```

In console output attachment (inside allure-reports directory) writes "????????" instead of cyrillic text.

**Comment by yotsubakoiwai** [comment]
Hi @f-z-f did you manage to produce the report? i can't :(

**Comment by unickq** [comment]
Hi @f-z-f 
I’ll check that, thanks

**Comment by unickq** [comment]
Hi @f-z-f ,
Fixed in [1.0.6](https://www.nuget.org/packages/NUnit.Allure/1.0.6)

![image](https://user-images.githubusercontent.com/6696821/58007274-9b701900-7af2-11e9-9384-63026da5dfc5.png)

**Comment by f-z-f** [comment]
> Hi @f-z-f did you manage to produce the report? i can't :(
For some reason today it's not reproduce. Thanks for helping.
¯\_(ツ)_/¯

**Comment by unickq** [comment]
For some reason?
I made a fix in 1.0.6 for that lol 🎃
